pom.xml

```diff
@@ -208,6 +208,13 @@
       <artifactId>slf4j-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-jdk14</artifactId>
```

Comment:
Does `slf4j-simple` work here as well? I don't strongly prefer either dependency, but since we use `slf4j-simple` in most other places it would be more consistent.